### PR TITLE
[#2240] feat(server): Add the operations for the group

### PR DESCRIPTION
**Pull request by qqqttt123**
### What changes were proposed in this pull request?
Add the operations for the group

### Why are the changes needed?

Fix: #2240 

### Does this PR introduce _any_ user-facing change?
Yes, I will add the document in the later pr.

### How was this patch tested?
Add a new UT.

### CODE REVIEW

File: server/src/main/java/com/datastrato/gravitino/server/web/rest/ExceptionHandlers.java [hunk 1]
```diff
@@ -265,10 +271,10 @@ private static class UserExceptionHandler extends BaseExceptionHandler {
     private static final ExceptionHandler INSTANCE = new UserExceptionHandler();
 
     private static String getUserErrorMsg(
-        String fileset, String operation, String metalake, String reason) {
+        String user, String operation, String metalake, String reason) {
```
Comment (qqqttt123) [inline server/src/main/java/com/datastrato/gravitino/server/web/rest/ExceptionHandlers.java:274]: Fix the wrong word by the way.

File: server/src/test/java/com/datastrato/gravitino/server/web/rest/TestGroupOperations.java [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+        Response.Status.INTERNAL_SERVER_ERROR.getStatusCode(), resp3.getStatus());
+
+    ErrorResponse errorResponse2 = resp3.readEntity(ErrorResponse.class);
+    Assertions.assertEquals(ErrorConstants.INTERNAL_ERROR_CODE, errorResponse2.getCode());
+    Assertions.assertEquals(RuntimeException.class.getSimpleName(), errorResponse2.getType());
+  }
+
+  @Test
+  public void testGetGroup() {
+
```
Comment (jerryshao) [inline server/src/test/java/com/datastrato/gravitino/server/web/rest/TestGroupOperations.java:166]: Can you please remove this blank line?

Comment (qqqttt123) [reply server/src/test/java/com/datastrato/gravitino/server/web/rest/TestGroupOperations.java]: Done.

**Comment by jerryshao** [comment]
Can you rebase to fix the conflict?

**Comment by qqqttt123** [comment]
> Can you rebase to fix the conflict?

Done. Merged the main branch.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**